File: spring-boot/base-java-h2/src/main/java/com/se/example/springbootjavah2/repository/CustomerRepository.java
package com.se.example.springbootjavah2.repository;

import com.se.example.springbootjavah2.entities.Customer;
import com.se.example.springbootjavah2.entities.CustomerId;
import org.springframework.data.jpa.repository.JpaRepository;

public interface CustomerRepository extends JpaRepository<Customer, CustomerId> {
}

File: SpringBoot/springboot-kotlin-h2-sample/src/main/kotlin/com/se/example/springbootkotlinh2sample/repositories/CustomerRepository.kt
package com.se.example.springbootkotlinh2sample.repositories

import com.se.example.springbootkotlinh2sample.entities.Customer
import com.se.example.springbootkotlinh2sample.entities.CustomerId
import org.springframework.data.jpa.repository.JpaRepository

interface CustomerRepository : JpaRepository<Customer, CustomerId> {
}